client/src/components/ui/navigationbar/TopBar.svelte

```diff
@@ -21,18 +23,25 @@
         ? null
         : $allOffices[$dashboardState.currentOffice];
     $: officeName = maybeOfficeName ?? '';
+
+    let isLogoutHovered = false as boolean;
+    const logoutIcon = new URL('../../../assets/icons/carbon/logout-profile.svg', import.meta.url);

```

Comment:
I recommend creating a component called `logoutIcon.svelte` to hold the icon as a component like every other icon in the app.